src/components/Cards/MeetingCard/MeetingCard.tsx

```diff
@@ -1,87 +1,53 @@
 import React from "react";
 import styled from "@emotion/styled";
-import Card from "../Base/Base";
-import colors from "../../../styles/colors";
-import { fontSizes } from "../../../styles/fonts";
+import "@mozilla-protocol/core/protocol/css/protocol.css";
+
+const DateOverlay = styled.p`
+  position: absolute;
+  z-index: 1;
+  color: white;
+  font-size: 30px;
+  font-weight: bolder;
+  top: 75%;
+  left: 2%;
+`;
 
 type MeetingCardProps = {
+  cardLink: string;
   imgSrc: string;
+  imgAlt: string;
   imgOverlayText: string;
   committee: string;
-  tags: [string];
+  tags: string[];
   excerpt?: string;
 };
 
-const HeadingSection = styled.div`
-  margin: 0 0 2rem 0;
-`;
-
-const SectionTitle = styled.div`
-  padding-bottom: 0.4rem;
-  font-size: ${fontSizes.font_size_3};
-  color: ${colors.grey};
-`;
-
-const HeadingContent = styled.div`
-  font-size: ${fontSizes.font_size_5};
-  font-weight: bold;
-  color: ${colors.black};
-`;
-
-const BodySection = styled.div`
-  margin: 0 0 1rem 0;
-`;
-
-const SectionContent = styled.div`
-  font-size: ${fontSizes.font_size_4};
-  font-weight: bold;
-  color: ${colors.black};
-`;
-
-const SearchExcerpt = styled.div`
-  font-size: ${fontSizes.font_size_3};
-  color: ${colors.grey};
-`;
-
-const MeetingCard = (props: MeetingCardProps) => {
-  const tagString = props.tags.join(` • `);
+const MeetingCard = ({
+  cardLink,
+  imgSrc,
+  imgAlt,
+  imgOverlayText,
+  committee,
+  tags,
+  excerpt,
+}: MeetingCardProps) => {
+  const tagString = tags.join(` • `);
```

Comment:
Just curious, is there an html character code to use instead of a unicode character?

Reply:
There is `&bull;`, but we'd have to use [dangerouslysetinnerhtml](https://reactjs.org/docs/dom-elements.html#dangerouslysetinnerhtml) to create the markup for tagString. Which is fine I think.

Reply:
I vote we don't use dangerouslySetInnerHTML too 😅. So I'm going to go ahead and resolve this conversation.